old_version/assets/js/main.js

```diff
@@ -397,6 +401,17 @@ let skillArray = [];
 let j = 0;
 let acc = 0;
 
+function gettingSkills() {
```

Comment:
Maybe we could use ternaries again?

```
skillArray = dataCard.skills,
        j = skillArray.length,
        acc = skillArray.length
      ): skillArray = [];
```

Reply:
I tried before but I'm not sure about the ternary syntax in this case. 

(dataCard.skills) ?
    skillArray = dataCard.skills;
    j = skillArray.length;
    acc = skillArray.length;
  :
    skillArray = [];

EsLint asks me to change the first  ; to a  : and I tried to put in brackets the skill Array, j and acc but It doesn't work either...